github-actions/feature-request/action.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Manage Feature Requests'
+description: 'Manages feature requests based on community demand'
+author: 'Angular'
+inputs:
+  token:
+    description: 'A token for the feature request bot.'
+    default: ${{ github.token }}
+  minimum-unique-comment-authors-for-consideration:
+    description: 'Number of unique comment authors required to move a feature request to our consideration list.'
+    default: 10
+  minimum-votes-for-consideration:
+    description: 'Number of votes required to move a feature request to our consideration list.'
+    default: 20
+  feature-request-label:
+    description: 'Label used to distinguish feature request from other issues.'
+    default: 'feature'
+  in-backlog-label:
+    description: 'Label used to distinguish feature requests, which are already part of the backlog.'
+    default: 'in-backlog'
```

Comment:
>We can probably rename this config option to in-backlog-filter and specify the entire filter segment (e.g. milestone:Backlog) as a value.

I like this suggestion! I noticed that currently we have all the issues in the backlog. Is this expected?

Reply:
Yeah, that's expected (see 2nd part of my comment above). The idea of the current `Backlog` milestone is to say that these tickets are triaged (so no immediate processing is needed), all tickets that are not in that milestone should be looked at and either closed or added to the `Backlog` (by adding extra labels to an issue). With feature requests, the only label that is needed to consider it triaged is "feature" (i.e. once that label is applied, the ticket is added to the `Backlog` milestone by NgBot). 

I believe we should have a new label to tell the bot that the ticket was already processed or should be ignored.

Reply:
I think adding extra label will simplify things and will not require any changes to the current process (i.e., `Backlog` will still be only for triaged issues).

What about if we rename the `in backlog` label to `roadmap`? I'm also leaning towards continuing to use labels in the both rather than milestones for consistency and simplicity.

Reply:
> What about if we rename the in backlog label to roadmap?

I think that might send wrong message to the issue author :) Having the `roadmap` label might be perceived as "hey, your feature request is now a part of our roadmap (i.e. it's accepted and will be implemented)". I was thinking may be we can call it `status: triaged`?